### PR TITLE
Add reworked crash ping cookbook (fixes #363)

### LINKED ISSUE #363: Rewrite "working with crash pings" for bigquery
See: https://docs.telemetry.mozilla.org/cookbooks/crash_pings.html

It still refers to pyspark and the deprecated dataset API, we could easily express this as a set of SQL statements.

This is of interest to release management, among others.

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

File: src/cookbooks/crash_pings.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+```sql
+SELECT date(submission_timestamp) AS crash_date,
+       count(*) AS crash_count
+FROM telemetry.crash
+WHERE date(submission_timestamp) >= DATE_SUB(CURRENT_DATE(), INTERVAL 7 DAY)
+GROUP BY crash_date
+```
+
+You can see an example of this query, along with a simple graph in [this STMO query](https://sql.telemetry.mozilla.org/queries/67925/).
```
Comment (scholtzan) [inline src/cookbooks/crash_pings.md]: It looks like the graph is broken. X-axis doesn't plot the date.

File: src/cookbooks/crash_pings.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+       normalized_os AS os,
+       count(*) AS crash_count
+FROM telemetry.crash
+WHERE date(submission_timestamp) >= DATE_SUB(CURRENT_DATE(), INTERVAL 7 DAY)
+  AND normalized_channel='nightly'
+GROUP BY normalized_os,
+         crash_date
+```
+
+Here's [another STMO query](https://sql.telemetry.mozilla.org/queries/67927/) showing this in action .
```
Comment (scholtzan) [inline src/cookbooks/crash_pings.md]: Tiny nit:

```suggestion
Here's [another STMO query](https://sql.telemetry.mozilla.org/queries/67927/) showing this in action.
```

File: src/cookbooks/crash_pings.md (hunk 1 above)
Comment (wlach) [reply src/cookbooks/crash_pings.md]: Good catch! I tweaked the query but forgot to update the visualization.

**Comment by wlach** [comment]
Thanks for the fast review @scholtzan!

**Pull request merged**